.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Insert a final newline here.

You can also add this line to `"files.insertFinalNewline": true` to `.vscode/settings.json`